packages/react-composites/package.json

```diff
@@ -31,6 +33,7 @@
     "@azure/communication-calling": "1.1.0-beta.2",
     "@azure/communication-chat": "1.0.0",
     "@azure/communication-common": "1.0.0",
+    "@azure/communication-identity": "1.0.0",
```

Comment:
I guess we can put it in dev-dependencies?

Reply:
Good catch